core/src/main/java/feign/AsyncFeign.java

```diff
@@ -44,10 +45,17 @@
  * be done (for example, creating and submitting a task to an {@link ExecutorService}).
  */
 @Experimental
-public abstract class AsyncFeign<C> extends Feign {
```

Comment:
Loved this.

I was hating myself for having `asyncBuilder` but couldn't take the time to fix it, amazing job, thanks